app/controllers/exam_controller.py

```diff
@@ -0,0 +1,146 @@
+import json
+from app.models.exam import Exam
+from app.models.user import User
+from app import response, app, db
+from flask import request, jsonify
+from flask_jwt_extended import *
+import datetime
+
+def format_exam(exam):
+    return {
+        "id": exam.id,
+        "question": exam.question,
+        "correct_option": exam.correct_option,
+        "option_a": exam.option_a,
+        "option_b": exam.option_b,
+        "option_c": exam.option_c,
+        "option_d": exam.option_d,
+        "user_id": exam.user_id,
+        "material_id" : exam.material_id,
+        "created_at": exam.created_at,
+        "updated_at": exam.updated_at
+    }
+
+def get_all_exam():
+    try:
+        exams = Exam.query.all()
+        formated_exams = [format_exam(exam) for exam in exams]
+        return jsonify({"exams": formated_exams})
+    except Exception as e:
+        return jsonify({"error": f"An error occurred: {e}"}), 500
+
+def get_exam_by_id(id):
+    try:
+        GetExam = Exam.query.get(id)
```

Comment:
please use correct format for variable naming in python

instead of GetExam, it should be get_exam
but I think you should just name it as `exam` since it is not a method

Reply:
done